src/SectionTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import * as Types from './types';
+
+export const SectionTitle: React.FC<Types.sectionDataProps> = ({ sectionData }) => {
```

Comment:
We'll want this component to be editable by the user, so they can rename the section if desired